book/version_control.md

```diff
@@ -395,140 +336,102 @@ To handle a data breach, you should:
 
-Like issues, pull requests can have assignees that are working on them.
-You can also assign reviewers and tag (`@`) project collaborators as part of the discussion.
+In the second case, perhaps you want to reuse or adapt code from an existing project. If the changes that you wish to make to the code are not in line with the aim of the original project, then you might create a fork to contain these changes.
 
+Note that forks do not automatically synchronise with the original repo. This means that changes to the original repo, after you create a fork, need to be manually synchronised if you want to include them in your repo. When you would like to offer to contribute your changes to the original project (see [Pull Requests](pull-requests)), you should ensure that you synchronise your branch with any new changes first.
 
-### Project boards
+See the GitHub documentation for [instructions on forking a repo and keeping your fork up to date](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo) with a project and also [working with forks](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/working-with-forks).
 
-[Project boards](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-project-boards) offer project management features through a [Kanban board](https://en.wikipedia.org/wiki/Kanban_board) interface.
+### Other GitHub features
 
-These boards can be used to track assignment and progress of specific tasks.
-This is aided by linking tasks to specific issues and pull requests.
+[Project boards](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-project-boards) offer project management features through a [Kanban-style board](https://en.wikipedia.org/wiki/Kanban_board). These boards can be used to track assignment and progress of specific tasks. This is aided by linking tasks to specific issues and pull requests.
 
+[GitHub Pages](https://pages.github.com/) offers hosting of static web content, which can be useful for project documentation. Alternatively, [project Wikis](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-wikis) can be used to document your project using [Markdown](https://www.markdownguide.org/basic-syntax/).
 
 (continuous-integration)=
 ## Continuous integration
```

Comment:
this should be in the testing bit, with the only add on here xplaining github actions as that is key for analysts. In practice if you could explain for analysts how to painlessly set up github actions (or link them to a tutorial) that might improve the uptake of tests as having a third party CI that isn't on github adds another barier for analysts to add basic tests to their code. I think the message here should be 'hey you wrote some tests - Sweet! now lets make them run all the time, look how easy it is and it is also recommended as best practice'

Reply:
ah you did that below apologies

Reply:
in which case I would reframe this as general github actions rather than just CI. Maybe we do - Github Actions can do loads of things for you automatically, list a few good actions and then provide a case study example of testing like you did rather than calling this CI off the get go because CI isn't something that belongs to github, but if we can stress how easy it is to do it, then that would be fab.